grisafi/app.js

```diff
@@ -0,0 +1,269 @@
+var express= require('express');
+var app = express();
+var bodyParser= require('body-parser');
+app.use(bodyParser.json());
+
+const DIA_EN_MILISEGUNDOS = 1000*60*60*24;
+
+var libros = new Array();
+var socios = new Array();
+var prestamos = new Array();
+
+function Libro(titulo,cantidad,id){
+    this.id=id;
+    this.titulo=titulo;
+    this.cantidad=cantidad;
+    this.disponibles=function(){
+        var count=0;
+        for (const prest of prestamos) {
+            if (prest.idLibro==this.id){
+                count++;
+            }
+        }
+        return this.cantidad-count;
+    }
+}
+
+function Socio(nombre,id){
+    this.id=id;
+    this.nombre=nombre;
+}
+
+function Prestamo(id,idSocio,idLibro,dias){
+    this.id=id;
+    this.idSocio=idSocio;
+    this.idLibro=idLibro;
+    this.fechavencimiento=Date.now()+DIA_EN_MILISEGUNDOS*dias;
+}
+
+function librosPrestados(idLibro){
```

Comment:
If you create all name functions in English this function should be in English too

Reply:
i'll think about it

Reply:
Please, use only one language